src/main/java/com/sulphate/chatcolor2/commands/ChatColorCommand.java

```diff
@@ -809,10 +721,8 @@ public String getModifier(String str) {
         }
     }
 
-    private char[] getCurrentRainbowSequence() {
-        verifyRainbowSequence((String) MainClass.getUtils().getSetting("rainbow-sequence"), true);
-        String seqstr = (String) MainClass.getUtils().getSetting("rainbow-sequence");
-        return seqstr.toCharArray();
+    private static char[] getCurrentRainbowSequence() {
```

Comment:
Same as previous.

Reply:
The rationale for making this method private and static is largely similar to the one I stated above. In this case in particular, however, I think it wasn't necessary, because the method always was used from a non-static context. But declaring it static allows sharing it between instances and could acommodate new static usages more easily.

Reply:
Alright, however - there is only ever one instance of this class initialised (in the MainClass class), so this probably isn't necessary like you said. Not a big problem, doesn't really matter either way I suppose. You choose :)